apphub/semantic_segmentation/unet3d/3DUnetMRBrainSegmentation_Isensee.ipynb

```diff
@@ -0,0 +1,908 @@
ve8KLfmUebkKYp/vzum52Zm6DX2OVIAhQl+Vz6KxLQctNeyLyRJAkO7bkF2++mVJi/NzY2ht4nn0A/ihCj5vrxuk9ciPlk3vOZU8NTJhEi9GQkijg2olv/ar+toEPk5jWX6udVvygl+1Plm1ZtUjclNxmQCx0KnAP1KeuaqknK4he/86Ysq3FbwcUKharJqSuD9VhhweYo5vNer+cF8C1ETDfddime/cBDSNMIEVQIKPqFQ3VFUbyg0MoxhPYDf1diW9blrBwALIZJLAayOX37Oz/vfNSjzKP2cWhdVxKsoMZneomDpV/+3OZMOwO/LrUAUYOO49idiKFFTwWXkEAZYsx5H0bjSo5nfrKcXMjInj33we/gzE+/qSC4Kq/QAFCCVSj4LukW9xI1cpsPQbVvjk1dSIoTatkCMqVzYmLC+Z4ZsLjYmJKThRVn0jRTaABPo9HAxMQEXnrJl16IBPQn6cbJfMy+Pypsus4RhIQggzAyjjKxmZodciVJpk3HcTyM9oYz4QB5NhxLiLS/SrBpslXNNx9PrvVFUeTMYpxb239lApq9h21qP+hyaDabALKsR5Q21Sw/NTWF2dnZRczR8sFll11W2JT8y7mwhFifqQbMOVFmqKlBCVpGo8I1yCs039q2EiL9Xfuh7VhLAKHf77tLFyzRsO/l9Q21L49op6XMVSH7KU8HagW9YamTYqr+vsp95iTevh+93Ddt+xLe9/mlEnoTUj6+8n2+0NwsN9gYEyowPMvL/vJ3/s0tOhF4ukTBCmB5cKs/b77AW2S4ZfM7ukzk6PMoxs+yxLc49gOzqDwwoFXdUFZBoeasY7L91b2vl9kQP7RuHsFS4Dwp/en3++7ugJzX5GOt1WrYvHkznnrqqZLxnxysqPg4OTnpDZCaw/T0NJ566poCYVOzo5Vs//zun/cc87r+PmEcpSmHkcYS9VEbNjc7+W1r/c9+4CG87hMXDX8HSIw4F1Yq5HEfe/6ZH0WU0PWH2g+VBgk2oEERfXJy0gXwqabN+jQ/7moCb5/hRlDTLvtor5jUebUbVk2E1CQWEs70PfZBr77TiPAQ41a81LXRd8r6oHmFrYlOpX+fcasFyfcZhixV1ooQ3h8RfGa6VMj3SE7Ii35S7dtiGGfkMg76QXUkxLaujClJr9zaFK0OKwWTk5NoNBpoNBqub4q7odMWi50PC7zhijjvnxEGQmue42V4jfK+ZJ/i8xyKOJZd8pK/k+fQtkFXdFtZjZqMVfew+p3tPmO2Rd1zesaZ7/IdBjdTSyZDpvDHSzY2btzoeJzOGwUvuuteLqyZ3N0VVFBBBa8FUO2QuQrI4PRmJoWiEKhuBd9NZpl7FpuQ4O3v3DeyXv8dQK0dUaQWSl/AzM3aYW3b/xu5qmnx0eBa3i5FAZTtkTkSrKCr86lCPUHN4/1+39UdOlpl27DxJ7VaDa1WC1NTU6jX6+h2u85VwTmdm5vD8ePHC3O7FFhRTVp9rfzwCJAdeJrmmsENt9wDREUpzCYAsZqK/wGIyCcjgVqEt1q23+f8WWb2Vmk997tsuu1SbLrtUi/hAqVnNVmqpEvEDWnPmnxDpVdKgkQ4lRJPnDjhmYb0fXuhOc1KNJOudgDZrl273ByolGr7oZqlRkbrXx6vsjhAzcKat0IbVX/T59Zqof3Tm3fUMsE+sXwIN61GorhgtUMtt/n2y4f7KA+IsnuozJRptaelaW1lgUV+O3kZ/2/oPTvO/G8MXoWZ7YnYI8TOchJnnzy+RRmctlP0Va8UNJtNZ72i+4r5pScnJ10sQpoWL6iw94MvtE6Km/kYeZTNp6P6Tv5JoEdNQ0KBtqPvW0aajWXIDME5j/DM+x/0aBuQWxcBeEKLpXekc2qBtMczrSU3ZB0krWS71JapFRPYRzXPc07UPM+6lgNWVJNWBKKvaHJyEk8+eRXqdSth5Yv5pXv3OCGOk2mPUuRS3vBbgYkWD6ufRM9RFjDBtmhW9NtNvXdf94mLMkLdyok3jwho0Bfg+yM1YEiJPPtSZlIig1ZmzOfdbteT4LUeboLx8XGHpCqgrGReWgs7d+4sMD2Oy6bH1Mh2llOzOM8qc9NoXAQ3n25mS/zsuisUhUw/eQzL8H3dsIxBsOtIfLUBMRynxUHrp8+JKNvhEbUsmjZNiwEtxTFad5HGdCxsDl7MnqMg4TNJ3z/t91GvziTNyL6pthVFkbuP2vWj0GXSGr/tpdGJCipYeRjJpFc7mw6AQjadsTQzIdxxx9uH9WXlAatpD2uM/DKEVcmmM3Gpa3/sf6my6SwFpqamCloTwUrtlrha6djOMd+zGl8o4CSkIWi/tFzZuuhvtoxdD322EChTsnV3p2706mvfcbowJIvT9tnIVk+irA/hvbcYpugz5lD/Q9PVmmplljhEzqSKFPjLv/gPLumHBc7lxMQ1AIBjx47hsccew+23376Ifp4cMC0v4AdrNhoNNJtNvPShR7A5fcTdwwyEzN1+v/m8DC8zq9go07Z7gjz/ur9nQhYN/c3iSBHns78ZfcvN6Jtu2xkM8pufn3fnoQE/zTLLM38E/wfgBYApDaAQT1+21dBVIaFmzLZC8z8+Po5Wq4Xjx4+j2+0WrH7LFeW94j7pomkktDkjRFGGEPkmWjwRKeddSycsi4MUqZxJ7Tx6OhoXHEEEoH/2HOo/nCw1CQFFc1EZc9LyoTqCPSuZlNBzDURSjTCUDW0lk8nz/muCatRlR3/02SiGyd9D9Y9ijmV1LqbdUX0pK6fE8GTWcBGtlDxf6T0yCiwTts8XfjfffznDTpH6DNqDogDeuOC5XNj+YYaD69atw5YtWxbRj5cHmu2OufXn05Br4ORxLH/Hz+mgilPYAllkmP7+KPqgR+8h0he/z7RGapAkfdI22Yi9K56WSA2yVVeojZJnuyxjlSAbrKd0mS5DBbbTbDZd0ieNFE+SZNmUmZFMeqnZdDQDFG34rVYLZ5xxBp5++i0Fc6ZOHs3dURwhkjtoeRbaat
```

Comment:
It would probably be good to have some kind of commentary at the end here about whether the inference result is actually good or not

Reply:
Inference section  is much detailed now with  dice loss for 'while tumor', 'tumor core' and 'enhancing tumor' . Added a visualization section as well.